ebos/eclproblem.hh

```diff
@@ -2943,6 +2950,44 @@ private:
         return dtNext;
     }
 
+    void computeAndSetEqWeights_() {
+        std::vector<Scalar> sumInvB(numPhases, 0.0);
+        const auto& gridView = this->gridView();
+        ElementContext elemCtx(this->simulator());
+        auto elemIt = gridView.template begin</*codim=*/0>();
+        const auto& elemEndIt = gridView.template end</*codim=*/0>();
+        for (; elemIt != elemEndIt; ++elemIt) {
+            const auto& elem = *elemIt;
+            if (elem.partitionType() != Dune::InteriorEntity)
+                continue;
+
```

Comment:
BTW There is a shortcut for this in modern DUNE that simplifies code:
```
for(const auto& elem: elements(gridView, Dune::Partitions::interior))

Reply:
Thanks. Changed. Who doesn't want to be modern!